Makefile

```diff
@@ -44,6 +44,9 @@ K8S_NAMESPACE ?= node-feature-discovery
 
 OPENSHIFT ?=
 
+RUN_FULL_E2E ?= true
```

Comment:
I'd suggest to prefix the name with E2E, like the other corresponding e2e-related variables, `E2E_TEST_ALL_IMAGES` or `E2E_TEST_FULL_IMAGE` or so... If we want we could also make this disabled by default (and enable it only in `scripts/test-infra/test-e2e.sh`) making local testing nicer – this could be done in a separate PR, too.

I think it would also make sense to move this a few lines downwards where the other `E2E_*` variables are specified.